styles/python.md

```diff
@@ -234,27 +204,30 @@ Because Python's indentation style is unlike many C-based languages, your editor
 Examples of line splitting from our code
 
 ```py
-BAD:  zero_duration_videos = video_models.Video.all().filter("duration =", 0).fetch(10000)
+# BAD
+
+zero_duration_videos = video_models.Video.all().filter("duration =", 0).fetch(10000)
 
-GOOD: zero_duration_videos = (video_models.Video.all()
+# GOOD
+zero_duration_videos = (video_models.Video.all()
                         .filter("duration =", 0)
                         .fetch(10000))
 ```
 
 ```py
-# BAD:
+# BAD
 return current_app.response_class("OAuth error. %s" % e.message, status=401, headers=build_authenticate_header(realm="http://www.khanacademy.org"))
 
-# GOOD:
+# GOOD
 return current_app.response_class(
     "OAuth error. %s" % e.message, status=401,
     headers=build_authenticate_header(realm="http://www.khanacademy.org"))
 ```
 
 ```py
-# Bad:
+# BAD
 kwargs = dict((str(key), value) for key, value in topic_json.iteritems() if key in ['id', 'title', 'standalone_title', 'description', 'tags', 'hide'])
-# Good:
+# GOOD
 kwargs = dict((str(key), value)
```

Comment:
According to the [Google style guide](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Indentation#Indentation) the first line should not contain arguments. Also, I'm not a fan of aligning with the open paren/bracket and prefer 4 (or multiple of 4) spaces (indenting by arbitrary spaces often requires more key-strokes, and if the open paren/bracket location changes because of a function name change, for instance, you have to update the alignment on the other lines too). What do others think?